### DISCUSSION TITLE
VSCodium :)

### DISCUSSION THREAD

**Discussion by IceMasterT** [docker/extension-ideas#183] [Ideas]
VSCodium is a Visual Studio that isnt made by MS. A lot of people dislike MS and refuse to use anything MS. It is frustrating for a team when issues like that come up. Well now there is a solution. It would be cool to have it on the Docker as an official thing. There is more of a why on the VSCodium Github. If this has been posted already, ooops; i didn't find one when i searched it out and felt id just toss it out there. Thanks Fellas! Cheers!
links---v
https://github.com/VSCodium
https://github.com/VSCodium/vscodium.github.io
https://github.com/VSCodium/vscodium-docker-files
https://vscodium.com/

**Comment by ajeetraina** [comment]
Did you check https://github.com/marcelo-ochoa/coder-docker-extension?
